sdk/core/core-amqp/src/cbs.ts

```diff
@@ -97,8 +98,7 @@ export class CbsClient {
             const id = context.connection.options.id;
             const ehError = translate(context.session!.error!);
             log.error(
-              "[%s] An error occurred on the session for request/response links " +
-                "for $cbs: %O",
+              "[%s] An error occurred on the session for request/response links " + "for $cbs: %O",
```

Comment:
Looks like a formatting change, but if the string can fit on one line we can combine these 2 into 1.